hometask1_1-java_14-features/src/test/java/ui/SauceTest.java

```diff
@@ -0,0 +1,109 @@
+        loginWithStandardUser();
+        assertEquals(BASE_URL.getValue() + INVENTORY_URL.getValue(), getWebDriver().getCurrentUrl());
+    }
+
+    @Test
+    @DisplayName("Navigation menu options test")
+    public void checkNavigationMenuOptions() {
+        loginWithStandardUser();
+        PAGE_HEADER.getNavigationMenuButton().click();
+        ElementsCollection menuOptions = PAGE_HEADER.getNavigationMenuOptions();
+
+        LOGGER.info("Checking list of links");
+        assertEquals(menuOptions.size(), expectedMainMenuOptionsMap.size());
+        menuOptions.forEach(optionLink -> {
+            assertTrue(expectedMainMenuOptionsMap.containsKey(optionLink.attr("id")));
+            assertTrue(expectedMainMenuOptionsMap.containsValue(optionLink.text()));
+        });
+    }
+
+    @Test
+    @DisplayName("Null pointer exception test")
+    public void checkNullPointer() {
+        loginWithStandardUser();
+        PAGE_HEADER.getCartButton().click();
+        ElementsCollection filteredList = null;
+
+        LOGGER.info("Provoke NPE");
+        assertTrue(filteredList.get(0).$(".inventory_item_price").exists());
+    }
+
+    @ParameterizedTest
+    @DisplayName("Switch-case test")
+    @CsvSource({
+        "Open,Open,true",
+        "Awaiting reply from customer,Customer replied,false",
+        "Awaiting reply from customer,Awaiting reply from customer,false",
+        "Closed,Open,true",
+        "Closed,Closed,false",
+    })
+    public void assertCartItemStatusAfterQuantityUpdate(String initialStatus, String actualStatus,
+                                                        boolean isReopenEnabled) {
+        String expectedStatus = switch (initialStatus) {
+            case "Closed" -> {
+                if (isReopenEnabled) {
+                    yield "Open";
+                } else {
+                    yield initialStatus;
+                }
+            }
+            case "New", "Open", "Customer replied" -> initialStatus;
+            case "Awaiting reply from customer" -> "Customer replied";
+            default -> throw new IllegalArgumentException("Invalid data is provided");
+        };
+
+        assertEquals(expectedStatus, actualStatus);
+    }
+
+    @Test
+    @DisplayName("Switch-case ui test")
+    public void checkSocialLinks(){
+        loginWithStandardUser();
+        PAGE_FOOTER.getSocialLinks().forEach(this::assertSocialLink);
+    }
+
+    private void assertSocialLink(SelenideElement link) {
+        String expectedLink = switch(Objects.requireNonNull(link.getAttribute("class"))){
+            case "social_twitter" -> "https://twitter.com/saucelabs";
+            case "social_facebook" -> "https://www.facebook.com/saucelabs";
+            case "social_linkedin" -> "https://www.linkedin.com/company/sauce-labs/";
+            default -> throw new InputMismatchException("Unexpected class provided");
+        };
+        assertEquals(expectedLink, link.$("a").getAttribute("href"));
+    }
```

Comment:
we don't need to store such methods in "TestRunner" class.
Think, where could you move it